brave-lists/debounce.json

```diff
@@ -413,12 +415,13 @@
   },
   {
     "include": [
-      "https://fmarier.github.io/brave-testing/debouncer/*"
+      "https://dev-pages.brave.software/navigation-tracking/*",
+      "https://dev-pages.bravesoftware.com/navigation-tracking/*"
     ],
     "exclude": [
     ],
     "prepend_scheme": "https",
     "action": "regex-path",
-    "param": "^/brave-testing/debouncer/(.*);.html$"
+    "param": "^/navigation-tracking/(.*);.html$"
   }
-]
+]
```

Comment:
I'd leave the newline there just in case one of the many parsers that reads this file has a problem with this.